bindings/core/src/method/account.rs

```diff
@@ -33,10 +33,9 @@ use serde::{Deserialize, Serialize};
 #[serde(tag = "name", content = "data", rename_all = "camelCase")]
 pub enum AccountMethod {
     /// Burn outputs.
-
-    /// When burn **native tokens**. This doesn't require the foundry output which minted them, but will not increase
-    /// the foundries `melted_tokens` field, which makes it impossible to destroy the foundry output. Therefore it's
-    /// recommended to use melting, if the foundry output is available.
+    /// When burning **native tokens**: This doesn't require the foundry output which minted them, but will not
```

Comment:
Maybe:
"Note that burning **native tokens** doesn't require the foundry output which minted them, ..."
is a bit easier to read.

Reply:
this is better. Fixed